### DISCUSSION TITLE
关于图标题内容超过一行就变成左对齐的问题

### DISCUSSION THREAD

**Discussion by fiona0417** [HFUTTUG/HFUT_Thesis#43] [General]
你好，
当图标题内容少于一行时，中英文图标题都会居中显示
但是当图标题内容超过一行时，中英文标题都会自动左对齐显示
请问，该如何解决

**Comment by LaureatePoet** [comment]
需要编辑 hfutthesis.cls#L3019-L3030
https://github.com/HFUTTUG/HFUT_Thesis/blob/6f0efdac293d749dbd65ce069cf97dbe19faebad/hfutthesis.cls#L3019-L3030
详细配置可以阅读[`caption`宏包](http://mirrors.ctan.org/macros/latex/contrib/caption/caption.pdf) Sec. 2.2

<img width="867" alt="image" src="https://user-images.githubusercontent.com/42093310/190584227-3e31904c-83be-4224-a884-b81fe577dae7.png">
